src/mesh/NodeDB.cpp

```diff
@@ -182,7 +182,11 @@ void NodeDB::installDefaultConfig()
 #else
     config.device.disable_triple_click = true;
 #endif
-    config.position.gps_enabled = true;
+#if HAS_GPS == 0
+    config.position.gps_mode = meshtastic_Config_PositionConfig_GpsMode_NOT_PRESENT;
+#else
+    config.position.gps_mode = meshtastic_Config_PositionConfig_GpsMode_ENABLED;
```

Comment:
Here I think we need to check for GPS_RX_PIN and config.position.rx_gpio. If neither is set then NOT_PRESENT.
Also #ifdef T_DECK then not_present.